build-config/resources/Ballerina.toml

```diff
@@ -2,7 +2,7 @@
 distribution = "2201.10.0"
 org = "ballerinax"
 name = "hubspot.crm.commerce.orders"
-version = "@toml.version@"
```

Comment:
This should not be changed. `@toml.version@` will be automatically replaced by the gradle script.